### PR TITLE
Tabnet on Vertex pipelines

### PR DESCRIPTION
**Pull request by udaypunna**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Cloud Storage bucket permission issues resolved. <br>
Textual corrections.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb [hunk 1]
```diff
@@ -96,8 +96,8 @@
       "source": [
```
Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>Vertex Pipelines -&gt; Vertex AI Pipelines</p><p> </p><p>Add to list: Vertex AI TabNet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>combine installs to a single pip3 install</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>Add one-liners on purpose of each helper function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>combine the 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>we -&gt; you</p><p> </p><p>For a complete list of supported feature transformation configs and examples, please go&nbsp;<a href="https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.15/google_cloud_pipeline_components.experimental.automl.tabular.html#google_cloud_pipeline_components.experimental.automl.tabular.FeatureTransformEngineOp" target="_blank">here</a>. -&gt;</p><p> </p><p>Learn more about [supported feature transformation configs and examples](link)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>For a full list of hyperparameters available for tuning, see&nbsp;<a href="https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.23/google_cloud_pipeline_components.experimental.automl.tabular.html#google_cloud_pipeline_components.experimental.automl.tabular.utils.get_tabnet_trainer_pipeline_and_parameters" target="_blank">here</a>. -&gt;</p><p>Learn more about [hyperparameters available for tuning](link)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (udaypunna) [reply notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>Updated to single pip3.</p>

Comment (udaypunna) [reply notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>Changed to Vertex AI SDK</p>

Comment (udaypunna) [reply notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>added one line comment for each helper function.</p>

Comment (udaypunna) [reply notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>Combined.</p>

Comment (udaypunna) [reply notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>Updated.</p>

Comment (udaypunna) [reply notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb:96]: <p>Updated.</p>

**Comment by andrewferlitsch** [comment]
313 self.submit(
Step #5: Step #4:     314     service_account=service_account,
Step #5: Step #4:     315     network=network,
Step #5: Step #4:     316     create_request_timeout=create_request_timeout,
Step #5: Step #4:     317 )
Step #5: Step #4: --> 319 self._block_until_complete()
Step #5: Step #4: 
Step #5: Step #4: File /workspace/workspace/env/lib/python3.9/site-packages/google/cloud/aiplatform/pipeline_jobs.py:464, in PipelineJob._block_until_complete(self)
Step #5: Step #4:     461 # Error is only populated when the job state is
Step #5: Step #4:     462 # JOB_STATE_FAILED or JOB_STATE_CANCELLED.
Step #5: Step #4:     463 if self._gca_resource.state in _PIPELINE_ERROR_STATES:
Step #5: Step #4: --> 464     raise RuntimeError("Job failed with:\n%s" % self._gca_resource.error)
Step #5: Step #4:     465 else:
Step #5: Step #4:     466     _LOGGER.log_action_completed_against_resource("run", "completed", self)
Step #5: Step #4: 
Step #5: Step #4: RuntimeError: Job failed with:
Step #5: Step #4: code: 9
Step #5: Step #4: message: "The DAG failed because some tasks failed. The failed tasks are: [exit-handler-1].; Job (project_id = python-docs-samples-tests, job_id = 7912992770569011200) is failed due to the above error.; Failed to handle the job: {project_number = 1012616486416, job_id = 7912992770569011200}"
Step #5: Step #4: 
Step #5: Step #4:

**Pull request merged**